pya/aserver.py

```diff
@@ -287,5 +292,18 @@ def _play_callback(self, in_data, frame_count, time_info, flag):
     def stop(self):
         self._stop = True
 
-    def __del__(self):
+    def _terminate_backend(self):
```

Comment:
this wrapping does not achieve much since self.backend is public. Why not call terminate directly?